app/models/scheme.rb

```diff
@@ -5,6 +5,7 @@ class Scheme < OpenStruct
   extend GdsApi::Helpers
 
   FACET_KEYS = [:areas, :business_sizes, :locations, :sectors, :stages, :support_types]
+  WHITELISTED_AREA_CODES = ["EUR", "CTY", "DIS", "LBO", "LGD", "MTD", "UTA"]
```

Comment:
Are we expecting to keep this in sync with [Publisher](https://github.com/alphagov/publisher/blob/master/app/models/area.rb#L7)? It might be worth a comment to reference it